docs/source/tools/trigger-service/index.rst

```diff
@@ -14,120 +14,139 @@ Trigger Service
 
 The :ref:`running-a-no-op-trigger` section shows a simple method using the ``daml trigger`` command to arrange for the execution of a single trigger. Using this method, a dedicated process is launched to host the trigger.
 
-Complex workflows can require running many triggers for many parties and at a certain point, use of ``daml trigger`` with its process per trigger model becomes unwieldy. The Trigger Service provides the means to host multiple triggers for multiple parties running against a common ledger in a single process and provides a convenient interface for starting, stopping and monitoring them.
+Complex workflows can require running many triggers for many parties and at a certain point, use of ``daml trigger`` with its process-per-trigger model becomes unwieldy. The trigger service provides the means to host multiple triggers for multiple parties running against a common ledger in a single process and provides a convenient interface for starting, stopping and monitoring them.
```

Comment:
I think "Trigger Service" is a proper name, but either way.

Reply:
I mostly wanted to make this consistent and the lower amount of changes was if I changed trigger service to be non-capitalized than to make auth middleware capitalized. But I'll change this to go the other way around.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/13111/commits/48a65754ee5e324cf2649d456c40ce990176c34e